galley/pkg/config/analysis/analyzers/schema/validation_test.go

```diff
@@ -35,10 +35,10 @@ func TestCorrectArgs(t *testing.T) {
 
 	m1 := &v1alpha3.VirtualService{}
 
-	testSchema := schemaWithValidateFn(func(name, ns string, msg proto.Message) (errs error) {
-		g.Expect(name).To(Equal("name"))
-		g.Expect(ns).To(Equal("ns"))
-		g.Expect(msg).To(Equal(m1))
+	testSchema := schemaWithValidateFn(func(cfg config.Config) (errs error) {
```

Comment:
Is there any way we could just get rid of `Config` and `Meta` and just use the [resource](https://github.com/istio/istio/blob/master/pkg/config/resource/) package?

Reply:
Discussed offline, we probably can at some point but this PR is large enough